src/middlewares/auth.middleware.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const user = require('../models/user.model');
```

Comment:
Just use import, instead of require for code consistency.

Reply:
for reference https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import